frontend/src/utils/getOverflowMenuButtons/getOverflowMenuButtons.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactElement } from 'react';
+
+/**
+ * Returns all of the buttons to be displayed in the toolbar overflow menu.
+ * @param {Array<ReactElement | false>} buttons - All of the buttons that could be rendered
```

Comment:
Do we need to overload with `false`?  Can we convey the same context with an empty array instead?

Reply:
I think we do. The `toolbarButtons` array has a conditional render, which returns `false` when we don't have report issue enabled.